test/dlc_tests/conftest.py

```diff
@@ -114,7 +114,8 @@ def ec2_instance(
         region = P3DN_REGION
         ec2_client = boto3.client("ec2", region_name=region, config=Config(retries={"max_attempts": 10}))
         ec2_resource = boto3.resource("ec2", region_name=region, config=Config(retries={"max_attempts": 10}))
-        ec2_instance_ami = UBUNTU_16_BASE_DLAMI_US_EAST_1
+        if ec2_instance_ami == UBUNTU_16_BASE_DLAMI_US_WEST_2:
```

Comment:
why is this line needed?

Reply:
This test is using a specific AMI https://github.com/YYStreet/deep-learning-containers/blob/pt/test/dlc_tests/benchmark/ec2/pytorch/training/test_performance_pytorch_training.py#L24

The origin logic is to forcefully apply `UBUNTU_16_BASE_DLAMI_US_EAST_1` to any tests running in us-east-1 and ignore the arg of AMI, which is incorrect

Reply:
This condition is confusing. It checks if the AMI is us-west-2, and then assigns a us-east-1 AMI instead. Since we expect default to be us-east-1  - it's better to assign the default unless the AMI is the same as PT AMI.